src/app/core/admin/admin-entity/admin-entity.component.ts

```diff
@@ -86,6 +85,10 @@ export class AdminEntityComponent implements OnInit {
     });
   }
 
+  updateCheckboxState(checkboxStatus: boolean) {
+    this.checkboxState = checkboxStatus;
```

Comment:
why do you need to interact with checkbox PII status in the parent component? This should be completely encapsulated within AdminGeneralSettingsComponent only and saved via the configEntitySettings together with other config.

--> see/map to EntityConfig.hasPII

Reply:
Previously  am using a custom checkbox to open the PII configuration, But now I know that we can use has PII from entity, So will change all occurrences.